save-plugins/fix-and-warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fixandwarn/FixAndWarnPluginConfig.kt

```diff
@@ -49,6 +54,11 @@ data class FixAndWarnPluginConfig(
         return FixAndWarnPluginConfig(
             fix.validateAndSetDefaults(),
             warn.validateAndSetDefaults()
-        ).also { it.configLocation = this.configLocation }
+        ).also {
+            it.configLocation = this.configLocation
+        }
+    }
+    companion object {
+        internal val defaultIgnoreLines: MutableList<Regex> = mutableListOf()
```

Comment:
Do we really need an extra constant for it? May be you will simply set the default value (`mutableListOf()`) in place, were it is created?

Reply:
To my mind it looked nicer. But ok, it seems that we don't need it here.